app/views/core/RootView.coffee

```diff
@@ -124,6 +124,9 @@ module.exports = class RootView extends CocoView
 
     if title = @getTitle() then title += ' | CodeCombat'
     else title = 'CodeCombat - Learn how to code by playing a game'
+    
+    if localStorage?.showViewNames
+      title = @constructor.name
```

Comment:
You can set it in the console and it'll stick around across users for that browser: `localStorage.showViewNames = true`

I also have a tampermonkey script to do it for incognito windows.